www/js/splash/updatecheck.js

```diff
@@ -45,6 +45,10 @@ angular.module('emission.splash.updatecheck', ['angularLocalStorage'])
          'Ionic Deploy: Update error! '+ res);
     }, function(prog) {
       console.log('Ionic Deploy: Progress... ', prog);
+      $rootScope.progress = prog;
+      $rootScope.isDownloading = true;
+    }).then(function(){
+      $rootScope.isDownloading = false;
```

Comment:
shouldn't this be in the existing `then` implementation above? I don't think that the implementation above returns a promise, does it?